File: CTProyecto/app/src/main/java/pe/edu/upeu/ctproyecto/data/model/Servicio.kt
package pe.edu.upeu.ctproyecto.data.model

import com.google.gson.annotations.SerializedName

data class Servicio(
    @SerializedName("id_servicio") val id: Int,
    val id_emprendimiento: Int,
    val nombre_servicio: String,
    val descripcion: String?,
    val precio: Double,
    val ubicacion: String?,
    val fecha_inicio: String?,
    val fecha_fin: String?,
    val estado: String
)

File: WebTurismo/app/Models/Servicio.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Servicio extends Model
{
    use HasFactory;

    protected $table = 'servicios';
    protected $primaryKey = 'id_servicio';
    public $timestamps = false;

    protected $fillable = [
        'id_emprendimiento',
        'nombre_servicio',
        'descripcion',
        'precio',
        'ubicacion',
        'fecha_inicio',
        'fecha_fin',
        'estado',
        'fecha_publicacion'
    ];

    // Relación con emprendimiento
    public function emprendimiento()
    {
        return $this->belongsTo(Emprendimiento::class);
    }
}
